### DISCUSSION TITLE
coil files

### DISCUSSION THREAD

**Discussion by jbusejackson** [simnibs/simnibs#387] [2. General]
Hi simnibs team, 

We plan to do some e-field modelling to adjust the intensity of stimulation from motor cortex to different brain regions. Although I couldn't see it in the list, is there a coil file available for 70BF-LQC? If not then how much do you think it would
change the ratio adjustment to match e-fields across different region using a different coil file?  Do you have a similar coil file to this one? 

Failing this, there appear to be a couple of threads about scripts to create a coil file and that these are in development. Is it possible to access this code, what information would I need to create a coil file?

Thanks and best wishes!

**Comment by Kstoffer** [comment]
We do not have access to this coil or detailed information about the windings available. However, judging from pictures it looks like it could be highly similar the the normal 70BF perhaps you can ask Daymed if the geometry is indeed identical to the normal 70BF.
Best,
Kristoffer
